packages/core/src/components/VerticalNavigation/Navigation/Navigation.tsx

```diff
@@ -7,7 +7,7 @@ import {
   ComponentProps,
 } from "react";
 import { clsx } from "clsx";
-import uniqueId from "lodash/uniqueId";
+import { uniqueId } from "lodash";
```

Comment:
Importing `lodash` like this will being in the entire bundle - we don't want that

```suggestion
import uniqueId from "lodash/uniqueId";
```